### PR TITLE
Make stubs for neonatal preterm and LBWSG

### PR DESCRIPTION
**Pull request by NathanielBlairStahn**

### CODE REVIEW

**Comment by NathanielBlairStahn** [comment]
> Is this just for completion of the NN causes, or did you decide you need preterm in the lsff model?

We don't need it in LSFF, just adding it for completion of the NN causes - it seems like it will make sense to document this at the same time we document the LBWSG risk.

**Comment by NathanielBlairStahn** [comment]
Figured I might as well add the related LBWSG risk since this PR is still open.

File: docs/source/models/gbd2017/risk_attributable_causes/neonatal_preterm/index.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+====================================
+
+Disease Description
+-------------------
+
+Case definition in GBD 2017
++++++++++++++++++++++++++++
+
+Preterm birth is defined as live birth before 37 completed weeks of gestation.
+In our analysis, we further break down this cause into three sub-categories of
```
Comment (beatrixkh) [inline docs/source/models/gbd2017/risk_attributable_causes/neonatal_preterm/index.rst]: Maybe make clear that the 'our' is GBD, as opposed to us?

File: docs/source/models/gbd2017/risks/low_birthweight_short_gestation/index.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+attributable burden due to each risk factor, the GBD definition of “low birth
+weight” therefore refers to all birth weight below the Theoretical Minimum Risk
+Exposure Level (TMREL) for birth weight. Likewise, new-borns have been typically
+been classified into gestational age (GA) categories of “extremely preterm” (<28
+weeks of gestation), “very preterm” (28-<32 weeks of gestation), and “moderate
+to late preterm” (32-<37 weeks of gestation). “Short gestation” in GBD refers to
+all gestational ages below the gestational age TMREL.
+
+Exposures and relative risks for the GBD Low birth weight and short gestation
+risk factors are divided into joint 500-gram birth weight and 2-week gestational
```
Comment (collijk) [inline docs/source/models/gbd2017/risks/low_birthweight_short_gestation/index.rst]: For actual breakdown see:
[Doc - Apr 12 2019 - 9-04 AM.pdf](https://github.com/ihmeuw/vivarium_research/files/3985988/Doc.-.Apr.12.2019.-.9-04.AM.pdf)

Comment (NathanielBlairStahn) [reply docs/source/models/gbd2017/risks/low_birthweight_short_gestation/index.rst]: @collijk Should I include a link to that image in the documentation?

File: docs/source/models/gbd2017/risk_attributable_causes/neonatal_preterm/index.rst (hunk 1 above)
Comment (NathanielBlairStahn) [reply docs/source/models/gbd2017/risk_attributable_causes/neonatal_preterm/index.rst]: Good point - I literally just copied and pasted this from the YLD Appendix. I'll make some minor edits to clarify.

**Pull request merged**